agent/bench-scripts/pbench-uperf

```diff
@@ -24,12 +24,16 @@ pbench_bin="`cd ${script_path}/..; /bin/pwd`"
 # source the base script
 . "$pbench_bin"/base
 
-benchmark_rpm=$script_name
 benchmark="uperf"
+benchmark_rpm=${benchmark}
 if [[ -z "$benchmark_bin" ]]; then
     benchmark_bin=/usr/local/bin/$benchmark
 fi
-ver=1.0.4
+ver="$(getconf.py version uperf)"
```

Comment:
This will have to be updated to use pbench-config after br69 branch moves into master